content/millennium/r4/entities/organization.md

```diff
@@ -36,10 +36,13 @@ Search for Organizations that meet supplied query parameters:
 
 ### Parameters
 
- Name        | Required?           | Type      | Description
--------------|---------------------|-----------|----------------------------------------------------------
- `_id`       | This or 'identifier'| [`token`] | The logical resource id associated with the resource.
- `identifier`| This or '_id'       | [`token`] | NPI, OID or FederalTAXID identifier for the organization.
+ Name        | Required?                                    | Type       | Description
+-------------|----------------------------------------------|------------|----------------------------------------------------------
+ `_id`       | This or 'identifier'                         | [`token`]  | The logical resource id associated with the resource.
+ `identifier`| This and/or any other search param, or `_id` | [`token`]  | NPI, OID or FederalTAXID identifier for the organization.
+ `type`      | This and/or any other search param, or `_id` | [`token`]  | The organization's type. Example: `http://terminology.hl7.org/CodeSystem/organization-type|govt`
```

Comment:
In the `Required?` column can we update it to something like this?
Line 42: This or type or _id
Line 43: This or identifier or _id

We shouldn't say 'and' here because It is not required to send both type and Identifier at the same type.

Reply:
Updated: https://github.com/cerner/fhir.cerner.com/pull/525/commits/b61cefae370edd3e2405672970dd61e130ab5b6c#diff-b7fb34600e684732b8db66567f7f6b6135899032c68460ff3ae42b2b19514f4bR41-R43